### PR TITLE
Added pin icon behind user marker images.

### PR DESCRIPTION
**Pull request by justinryder**
Based on the suggestion in [this post](https://www.funfunforum.com/t/show-us-where-you-are-fff-user-map/1659/75), I added back the marker icon behind the user marker profile images.

### CODE REVIEW

File: src/services/user-map-service.js [hunk 1]
```diff
@@ -18,14 +18,13 @@ const defaultIcon = icon({
...
-  icon({
-    iconRetinaUrl: `https://cdn-standard6.discourse.org/user_avatar/www.funfunforum.com/${username}/240/1285_1.png`,
-    iconUrl: `https://cdn-standard6.discourse.org/user_avatar/www.funfunforum.com/${username}/90/149_1.png`,
+  divIcon({
     shadowUrl: null,
     popupAnchor: [0, -20],
     iconSize: [28, 28],
     iconAnchor: [14, 28],
     className: 'marker-user-icon',
+    html: `<img class="marker-user-icon__image" width="28" height="28" src="https://cdn-standard6.discourse.org/user_avatar/www.funfunforum.com/${username}/90/149_1.png" />`,
```
Comment (Vazerthon) [inline src/services/user-map-service.js]: Could you remove `width="28" height="28"` from here as it's being set in CSS

**Review by Vazerthon — CHANGES_REQUESTED** [review]
Looks really cool. If you could take the width and height out of the `img` tag please that'd be ace :-)

File: src/services/user-map-service.js [hunk 2]
```diff
@@ -18,14 +18,13 @@ const defaultIcon = icon({
 });
 
 const userIcon = username =>
-  icon({
-    iconRetinaUrl: `https://cdn-standard6.discourse.org/user_avatar/www.funfunforum.com/${username}/240/1285_1.png`,
-    iconUrl: `https://cdn-standard6.discourse.org/user_avatar/www.funfunforum.com/${username}/90/149_1.png`,
+  divIcon({
     shadowUrl: null,
```
Comment (Vazerthon) [inline src/services/user-map-service.js]: Just realised, we can also re-enable the default shadow now

**Comment by justinryder** [comment]
Good catch, I forgot to remove those attributes once I got the wrapping div sized properly. Updated!

I looked into re-enabling the shadows since you mentioned it, but [the divIcon doc](http://leafletjs.com/reference-1.2.0.html#divicon) says shadow options are ignored. :(

**Review by Vazerthon — APPROVED** [review]
🎉 Awesome work!

**Pull request merged**